Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -73,149 +79,157 @@ private void Start()
+                float frameHeight = 80;
+                if (TwineNodeList.Count() > 0){
+                    frameHeight = Math.Min(Screen.height / (TwineNodeList.Count()), 80);
                 }
+                int index = TwineNodeList.IndexOf(this);
+                Rect frame = new Rect (10, 10+index*80, frameWidth, frameHeight);
+                GUIStyle style = new GUIStyle (GUI.skin.box);
+                style.wordWrap = true;
+                style.fixedWidth = frameWidth-10;
+                GUI.BeginGroup(new Rect(20, 10+index*frameHeight, frameWidth + 10, frameHeight));
+                scrollPosition = GUILayout.BeginScrollView(scrollPosition, GUILayout.Width(frameWidth + 10), GUILayout.Height(frameHeight));
+                content = content.TrimEnd();
+                GUILayout.Label(new GUIContent(this.content), style, GUILayout.Width(frameWidth - 20), GUILayout.ExpandHeight(true));
+                
+                if (isDecisionNode) {
+                    GUIStyle decisionHintStyle = new GUIStyle (style);
+                    decisionHintStyle.fontStyle = FontStyle.BoldAndItalic;
+
+                    if (!isOptionsGuiOpen) {
+                        GUILayout.Box ("Press TAB to progress in the story...", decisionHintStyle);
+                    } else {
+                        GUILayout.Box ("Press TAB to scroll, E to close, ENTER to choose", decisionHintStyle);
+                    }
+                }
+
+                if (this.isOptionsGuiOpen) {
+                    // Draw list of buttons for the possible children nodes to visit:
+                    GUIStyle optionButtonStyle = new GUIStyle (GUI.skin.button);
+                    optionButtonStyle.fontStyle = FontStyle.Italic;
+                    optionButtonStyle.wordWrap = true;
+
+                    // Set highlighted button to have white text (this state is called `onNormal`):
+                    optionButtonStyle.onNormal.textColor = Color.white;
+                    // Set non-highlighted buttons to have grayed out text (state is called `normal`)
+                    optionButtonStyle.normal.textColor = Color.gray;
+
+                    selectedOptionIndex = GUILayout.SelectionGrid(selectedOptionIndex, this.childrenNames, 1, optionButtonStyle);
+                }
+                GUILayout.EndScrollView();
+                GUI.EndGroup ();
+
             }
+            else if (this.enabled && !this.isMinimized && !allMinimized) {
+                float frameWidth = Math.Min(Screen.width / 3, 350);
+                float frameHeight = Math.Min(Screen.height / 2, 500);
+                Rect frame = new Rect (10, 10, frameWidth, frameHeight);
+                GUI.BeginGroup (frame);
+                GUIStyle style = new GUIStyle (GUI.skin.box);
+                style.wordWrap = true;
+                style.fixedWidth = frameWidth;
+                GUILayout.Box (this.content, style);
+
+                if (isDecisionNode) {
```

Comment:
Line 196-221 is exactly the same as line 158-183. 
Factor it out into a function or change the if condition arrangement so you don't have to copy this code.  Otherwise, it will be hard for future dev this way since you have to make sure you make changes to both places at the same time all the time.